src/App.tsx

```diff
@@ -1,10 +1,54 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Todo } from './types/Todo';
+import { FilterOption } from './types/FilterOption';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { Alert } from './components/Alert';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
 
-const USER_ID = 0;
+const USER_ID = 10527;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(FilterOption.ALL);
+  const [hasError, setHasError] = useState(false);
+  const [errorMessage] = useState('');
```

Comment:
```suggestion
  const [errorMessage, setErrorMessage] = useState('');
```
Where is setState function? How are you going to change the state?